docs/antrea-ipam.md

```diff
@@ -218,6 +218,34 @@ metadata:
 A StatefulSet Pod's IP will be kept after Pod restarts, when the IP is allocated from the
 annotated IPPool.
 
+### IPAM behaviors
+
+When `AntreaIPAM` is enabled, `antrea-controller` will watch IPPool CRs and StatefulSets
+from K8s apiserver, and `antrea-agent` will register IPAM driver to CNI server.
+
+#### On IPPool CR create/update event
+
+`antrea-controller` will update IPPool counters, and periodically clean up stale IP addresses.
+
+#### On StatefulSet create event
+
+`antrea-controller` will check annotations and preallocate IPs from the IPPool.
+
+#### On StatefulSet delete event
+
+`antrea-controller` will clean up IP allocations under this StatefulSet.
+
+#### On Pod create
+
+`antrea-agent` will receive a CNI add request, it will check annotations and allocate an IP from
+an IPPool. The IP is a pre-allocated StatefulSet IP, a specified IP or the next available IP in
+the IPPool.
+
+#### On Pod delete
+
+`antrea-agent` will receive a CNI del request and release the IP allocation from the IPPool.
+The pre-allocation status won't be changed.
```

Comment:
Do not understand this sentence. You probably should add more information.

Reply:
Changed to:
```
If the IP is a pre-allocated StatefulSet IP, it will remain pre-allocated status thus the Pod will
get same IP after recreated.
```